validator/enforce_labels_test.rego

```diff
@@ -414,3 +423,27 @@ test_enforce_label_bq_violations {
 	is_string(violation.msg)
 	is_object(violation.details)
 }
+
+#### Testing for Spanner Instances
+# Confirm 2 violation, one for the CAI representation and one for the API
+test_enforce_label_spanner_violations {
```

Comment:
Could we switch this to use the new test utils? https://github.com/forseti-security/policy-library/blob/master/validator/test_utils.rego

Reply:
Updated all tests to use the new library.